rocks/sackd/rockcraft.yaml

```diff
@@ -13,10 +13,10 @@
 # limitations under the License.
 
 name: sackd
-version: 25.05.1
-summary: The Slurm authentication and credential kiosk daemon of Slurm
+version: 23.11.4-1
```

Comment:
Hmm... not sure if we should pin to the Debian package version here since the rocks would ideally track the version of Slurm and not necessarily the version of the package we're pulling, especially if we switch to a PPA later down the line.

My concern is that we'd need to update the version of the OCI image everytime we pull a new version of the package, but we also don't pin the version of the Debian package we're pulling, so we'd really have no idea if we're actually pulling the Debian package version that we think we're pulling. If we just had the version set to the version of Slurm installed, we wouldn't have to update the version of the rocks each time we pull a new package with security fixes and the like.

Thoughts?

Reply:
Uhh I rolled back that change on a previous push 😅